fastfood/templating.py

```diff
@@ -57,7 +58,9 @@ def render_templates_generator(*files, **template_map):
                              % os.path.relpath(path))
         else:
             try:
-                template = JINJA_ENV.from_string(open(path).read())
+                with codecs.open(path, encoding='utf-8') as f:
```

Comment:
:+1: This aligns with utf-8 in Python 3. It's not clear to me that we weren't also doing this in Python 2.x, but at least it's explicit now.